types/fundamental-react/lib/Shellbar/Shellbar.d.ts

```diff
@@ -9,6 +9,7 @@ export type ShellbarProps = {
         notificationCount: number;
         callback?: (event: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void;
     }>;
+    backButton?: () => void;
```

Comment:
This was called `backAction` not `backButton` right?
and it looks like it does take an event, so it could be
```ts
onClick?: (event: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void;
```

Reply:
you are completely right, my mistake.